content/hi/docs/reference/glossary/group-version-resource.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Group Version Resource
```

Comment:
I'd expect to see Hindi  here; you can use `aka` to also show the English terms.